src/ArticleItem.jsx

```diff
@@ -0,0 +1,36 @@
+import { Link } from "react-router-dom";
+
+const ArticleItem = ({ article, className }) => {
+	return (
+		<div className={`articleListItem ${className}`}>
+			<img
+				className="articleListItemImage"
+				src={article.article_img_url}
+				alt={`${article.title}`}
+			/>
+			<div className="articleTitleContainer">
+				<Link
```

Comment:
I can see that this is prep for the next ticket - try not to leave any links to nowhere in your commits